app/assets/stylesheets/application.scss

```diff
@@ -89,9 +89,80 @@ body{
 
 // About Us page
 
-.linkedIn {
+.about-us-header {
+    margin-top: 90px;
+}
+
+.about-us-container{
+    background-color: #ddb57d;
+    border:dotted;
+    border-color: #342316;
+    margin-left: 50px;
+    margin-right: 50px;
+    padding-top: 35px;
+}
+
+.squares-container-left {
+    display: flex;
+    flex-direction: column;
+}
+
+.squares-container-right {
+    display: flex;
+    flex-direction: column;
+}
+
+.linked-in-right {
     max-height: 100px;
     max-width: 100px;
+    margin-left: 75%;
+}
+
+.linked-in-left {
+    max-height: 100px;
+    max-width: 100px;
+    margin-right: 75%;
+}
+
+.about-me-images-container {
+    height: 30vh;
+}
+
+.brandon-self-image {
```

Comment:
You might consider listing all your dev-self-image classes together to avoid repeat styling.
```
.brandon-self-image .simon-self-image .josh-self-image .jacob-self-image {
     ...
```